<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Routing</class>
 <widget class="QDialog" name="Routing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>921</width>
    <height>419</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>921</width>
    <height>419</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Routing</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>901</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>170</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Alias</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Antenna</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cable</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Probe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Relais-Matrix</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>edit Route</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>881</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbAntenna"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbCable"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbProbe"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbRelais"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>901</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="BtnNew">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>new</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnEdit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnAdd">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>add/update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>36</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="BtnCancel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnOk">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cBReceiver">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>FSWT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FSET</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ESIB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ESU</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BtnRoute">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>route and set Transducer of: -&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
